src/SupportMessage/Collection.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { FC } from 'react'
+import styled from 'styled-components'
+
+import { Text } from '../Text'
+import { Box } from '../Box'
+import { SupportMessage } from './SupportMessage'
+
+export const CollectionPage: FC = () => {
+  return (
+    <Box flex direction="column">
+      <ContainerBox>
+        <Label tag="span" typo="header-small">
+          Type info-outline
+        </Label>
+        <SupportMessage
+          type="info-outline"
+          description="Info support message"
+        />
+      </ContainerBox>
```

Comment:
Is it normal that `info-outline` don't have an example with a title?

Reply:
Yes doesn't have title example in designs + this is just displaying examples to user in Storybook